From 2b17f76f1495ee7eda50a754f8149e5ff3d04312 Mon Sep 17 00:00:00 2001
From: Otavio Salvador <otavio@ossystems.com.br>
Date: Wed, 13 Dec 2017 11:00:04 -0200
Subject: [PATCH] egl: Add EGL_PLATFORM_X11_KHR alias to EGL_PLATFORM_X11_EXT
Organization: O.S. Systems Software LTDA.

New EGL code uses EGL_PLATFORM_X11_KHR (for example glmark2) and the
value is the same, so we can provide an alias for it instead of
patching every code using it.

Upstream-Status: Pending

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
---
 gpu-core/usr/include/EGL/eglext.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gpu-core/usr/include/EGL/eglext.h b/gpu-core/usr/include/EGL/eglext.h
index d3c5f56..5f8d906 100644
--- a/gpu-core/usr/include/EGL/eglext.h
+++ b/gpu-core/usr/include/EGL/eglext.h
@@ -450,6 +450,7 @@ EGLAPI EGLSurface EGLAPIENTRY eglCreatePlatformPixmapSurfaceEXT (EGLDisplay dpy,
 #ifndef EGL_EXT_platform_x11
 #define EGL_EXT_platform_x11 1
 #define EGL_PLATFORM_X11_EXT              0x31D5
+#define EGL_PLATFORM_X11_KHR              EGL_PLATFORM_X11_EXT
 #define EGL_PLATFORM_X11_SCREEN_EXT       0x31D6
 #endif /* EGL_EXT_platform_x11 */
 
-- 
2.15.1

